chrome/browser/browser_resources.grd

```diff
@@ -656,6 +656,8 @@
       <if expr="not is_android">
         <include name="IDR_TAB_RANKER_EXAMPLE_PREPROCESSOR_CONFIG_PB" file="resource_coordinator\tab_ranker\example_preprocessor_config.pb" type="BINDATA" />
       </if>
+      <include name="IDR_ADBLOCK_ELEMHIDE_JS" file="../../third_party/libadblockplus/grit/resources/elemhide.js" type="BINDATA" />
+      <include name="IDR_ADBLOCK_ELEMHIDE_FOR_SELECTOR_JS" file="../../third_party/libadblockplus/grit/resources/elemhide_for_selector.jst" type="BINDATA" />
```

Comment:
.jst for for "JavaScriptTemplate". if we call is just ".js" "js_minify.py" script will be called and fails on packing of js because of `{{..}}` tokens used. Probably we can choose another replace token like /*url*/ to be js-valid